typings/blockly.d.ts

```diff
@@ -2130,58 +2114,40 @@ declare module "toolbox/separator" {
     }
 }
 declare module "utils/toolbox" {
-    /**
-     * -state: (*|undefined),
-     *            icons: (!Object<string, *>|undefined),
-     *            fields: (!Object<string, *>|undefined),
-     *            inputs: (!Object<string, !ConnectionState>|undefined),
-     *            next: (!ConnectionState|undefined)
-     *          }}
-     */
-    export type extra = {
```

Comment:
Interesting. Somehow `extra-state` got parsed into this. I guess hyphens do something weird.

Reply:
yup, there are a few other weird places too, empty comment strings in the ts defs where they are not empty in the js, missing strings entirely, etc. i only looked into the ones that actually caused errors though :/ for some reason there was a type declared called `Array` which caused an error that "Array is not generic" which was a very confusing error because the ts docs use Array as the example for how generics work.

it will be great when the types don't rely on having to parse out comments!